CHANGES.md

```diff
@@ -22,6 +22,7 @@ Release Notes.
 * Support [Native SQL](https://pkg.go.dev/net/http) database client framework with [MySQL Driver](github.com/go-sql-driver/mysql).
 * Support [Logrus](https://github.com/sirupsen/logrus) log report to the backend.
 * Support [Zap](https://github.com/uber-go/zap) log report to the backend.
+* Support [Native Mutex](https://pkg.go.dev/sync) lock client framework.
```

Comment:
```suggestion
* Support [Native Mutex](https://pkg.go.dev/sync) lock client framework.
```

Lock client? Is it a distributed lock solution as well?

Reply:
Also, you only support tracing, but not logic endpoint or metric of lock(pending, release, etc.).
Should we consider that?

Reply:
> Lock client? Is it a distributed lock solution as well?

It's a local lock solution. Not support distribution. 

> Also, you only support tracing, but not logic endpoint or metric of lock(pending, release, etc.).
> Should we consider that?

I think the metrics should be considered, but the original lock would be triggered by Go Self and frameworks. The metrics maybe difficult to understand.

Reply:
I mean, how to measure a lock performance? Usually, if lock pending time is long(lock method is blocked), it mean too many competition or not enough resources pooled.

Reply:
Yes, I think the lock time is the main metric for measuring a lock performance. We can intercept the lock begin time and finish time. 
Also, If it's using `TryLock` we can measure the failure lock count.

Reply:
Lock begin method should indicate the pending time before lock acquired, right?

At least 3 metrics, acquisition latency, using duration, error count(for try lock)

Reply:
Yes.

acquisition latency: getting lock used time. 
using duration: the duration between lock and unlock time.
error count: can't getting lock count(for `TryLock`).

Am I right?

Reply:
Yes, would this be listed as Golang VM tab?

Reply:
Yes, I could add to it.

Reply:
The metrics been added to this PR.

Reply:
The metrics been added to this PR.